antlr4ide.ui.console/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>antlr4ide.ui.console</name>
```

Comment:
I am not a great fan of this project name.
Consider either `antlr4ide.ui/console` as a subdir under UI, or a new two level name like `antlr4ide.log4j`